pkgs/development/php-packages/vld/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, buildPecl
+, stdenv
+, fetchFromGitHub
+}:
+
+let
+  version = "0.18.0";
+in buildPecl {
+  inherit version;
+
+  pname = "vld";
+
+  src = fetchFromGitHub {
+    owner = "derickr";
+    repo = "vld";
+    rev = version;
+    hash = "sha256-1xMStPM3Z5qIkrRGfCKcYT6UdF1j150nt7IleirjdBM=";
+  };
+
+  # Tests relies on PHP 7.0
+  doCheck = false;
+
+  meta = with lib; {
```

Comment:
I think we can remove the `with lib;` and sort alphabetically the `meta` attributes.